lua/neogit/buffers/rebase_editor/init.lua

```diff
@@ -63,6 +74,7 @@ function M:open(kind)
     filetype = "NeogitRebaseTodo",
     buftype = "",
     kind = kind,
+    disable_line_numbers = false,
```

Comment:
This could be set via config instead - there's already a "disable_line_numbers" value that could be read.

Reply:
here's my config,but i have no line number in rebase editor. is my config wrong?
```lua
opts={
      disable_line_numbers = false,
      rebase_editor = {
        disable_line_numbers = false,
      },
```
any way ,hard coded line number is still a bad idea,i will delete this commit.